lib/cretonne/meta/base/instructions.py

```diff
@@ -537,6 +537,21 @@
         ins=(x, src, dst),
         other_side_effects=True)
 
+copy_special = Instruction(
+        'copy_special', r"""
+        Copies a value from one special register to another. e.g. rbp -> rsp.
+        """,
+        ins=(src, dst),
+        other_side_effects=True)
+
+Offset = Operand('Offset', offset32, 'Offset from current stack pointer')
```

Comment:
I'm slightly confused about this one. I had thought about this and decided against it because the value should never be larger than 32-bit, and Intel's ADD operation doesn't support an imm64. Which would mean the generated code would need to mov the imm64 to a register and then add it, which seemed silly to me.

Should I maybe use an imm32 instead?